### ISSUE TITLE
Error: Chromium revision is not downloaded. Provide TAIKO BROWSER PATH or Install taiko again to download bundled chromium

### ISSUE THREAD

**Issue by shaifaliyadav** [getgauge/taiko#2235]
While trying to run test locally in my system, getting above error. It runs fine on remote server(docker)

**Comment by saikrishna321** [comment]
@shaifaliyadav Looks like taiko is unable to download chromium. Check project node_modules and taiko directory to see if chromium is downloaded.

**Comment by shaifaliyadav** [comment]
@saikrishna321 yes i's not getting downloaded, that is the problem. Tried installing/uninstalling it, but no luck

**Comment by shaifaliyadav** [comment]
this issue still exists..Can someone please help. @zabil @saikrishna321

**Comment by zabil** [comment]
Sorry to see that you are stuck with this issue but It's a bit hard to help with this as there is very little info to replicate it.

When creating a new issue there are a few sections to be filled up which helps find the actual problem. Those sections seem to be missing. Can you please fill up those sections? It has info about versions, environments, os, command to replicate it etc.

Without enough info a guess could be that there is a proxy for your local environment, if that is the case please check your [proxy settings](https://askubuntu.com/questions/583797/how-to-set-a-proxy-for-terminal)

**Comment by 0x12woo** [comment]
You can download it yourself in package
` node node_modules\taiko\lib\install.js`

**Comment by louislebrault** [comment]
having the same issue on Manjaro 21.2.0
